lib/block-supports/typography.php

```diff
@@ -538,7 +539,12 @@ function gutenberg_get_typography_font_size_value( $preset, $should_use_fluid_ty
 	 * the given font size multiplied by the min font size scale factor.
 	 */
 	if ( ! $minimum_font_size_raw ) {
-		$calculated_minimum_font_size = round( $preferred_size['value'] * $default_minimum_font_size_factor, 3 );
+		$preferred_font_size_in_px = 'px' === $preferred_size['unit'] ? $preferred_size['value'] : $preferred_size['value'] * 16;
+		// Logarithmic scale factor: Min font scale that tapers out as the font size increases.
+		$minimum_font_size_factor = 1 - 0.12 * log( $preferred_font_size_in_px );
```

Comment:
Please excuse my naivety here: where does the `0.12` value come from, is it just a pleasing value, or is it more significant?

Reply:
I agree, it feels a bit magic 😄 

It affects how quickly the curve will move towards the minimum, that is, how quickly the size factor reaches 0 given increasing font size values.